app/controllers/sales_logs_controller.rb

```diff
@@ -36,11 +37,34 @@ def edit
     end
   end
 
+  def download_csv
+    unpaginated_filtered_logs = filtered_logs(current_user.sales_logs, search_term, @session_filters)
+    codes_only = codes_only_export?(params)
+
+    render "download_csv", locals: { search_term:, count: unpaginated_filtered_logs.size, post_path: email_csv_sales_logs_path, codes_only: }
+  end
+
+  def email_csv
+    all_orgs = params["organisation_select"] == "all" # what's this for? params['organisation_select'] appears to always be nil
```

Comment:
is it one of the search filters? to restrict to all orgs, or a specific org?

Reply:
I need toi investigate this. Left it in as comment since I don't have the asnwer!